CreateCardForIssue/action.yml

```diff
@@ -9,4 +9,4 @@ inputs:
     required: true
 runs:
   using: 'node16'
-  main: 'CreateCardForIssue.js'
+  main: '../dist/CreateCardForIssue/CreateCardForIssue.js'
```

Comment:
This change also needs to be done for `AssignCardToSender` directory

Reply:
done https://github.com/SonarSource/gh-action-lt-backlog/pull/19/commits/76f32b1cb07512ff0d912e47028c63df377277d1